forest.js

```diff
@@ -6,12 +6,13 @@ const stubJoi = require('./lib/stub-joi');
 const call = new require('call');
 const router = new call.Router();
 
-module.exports = (server, opts, next) =>  {
+module.exports = {};
```

Comment:
is this necessary?

Reply:
would you like to indent the whole register function content (nearly the whole file) one more "tab"?

Reply:
Ok, "initializing" `module.exports` is not needed, as I didn't know. Removed this line.